Sources/GRPCCore/Call/Client/Internal/ClientStreamExecutor.swift

```diff
@@ -77,37 +25,52 @@ internal struct ClientStreamExecutor<Transport: ClientTransport> {
   ///   - request: A streaming request.
   ///   - method: A description of the method to call.
   /// - Returns: A streamed response.
-  @inlinable
-  func execute(
-    request: ClientRequest.Stream<[UInt8]>,
-    method: MethodDescriptor,
-    stream: RPCStream<Transport.Inbound, Transport.Outbound>
-  ) async -> ClientResponse.Stream<[UInt8]> {
-    // Each execution method can add work to process in the 'run' method. They must not add
-    // new work once they return.
-    defer { self._work.continuation.finish() }
-
-    // Start processing the request.
-    self._work.continuation.yield(.request(request, stream.outbound))
+  @inlinable  // would be private
+  static func execute<Input: Sendable, Output: Sendable>(
```

Comment:
Nit: do we want to rename to `_execute`?

Reply:
Ah thanks. We actually don't want it to be `_execute`, it should be `internal`, it's the comment that's wrong.